### PR TITLE
Emotes

### PR DESCRIPTION
**Pull request by Stingray540**
:cl: Stingray
rscadd: Adds a few funny emotes for Spartans, elites, and unggoy. Will probably add more once I find clean sounds for them. Open to suggestions!
/:cl:

### CODE REVIEW

File: code/modules/halo/emotes/emotes.dm [hunk 1]
```diff
@@ -11,11 +11,26 @@
 	. = ..()
 	species_sounds[/datum/species/sangheili] = oldref
 
+/decl/emote/audible/species_sound/aim
```
Comment (X0-11) [inline code/modules/halo/emotes/emotes.dm:14]: Could have a generalised *taunt that has a much wider array of sounds. I'm sure you'll find a bunch of 'em.

**Review by bloxgate — COMMENTED** [review]
What license are these sounds under? Currently all sounds and icons in the code need to be licensed CC 3.0 BY-SA, unless you include special documentation with them indicating otherwise.

**Comment by Stingray540** [comment]
James has claimed that 343 have given permission for these types of things, so I would assume that these are fine.

**Comment by bloxgate** [comment]
Please do leave a note that these are licensed under the game content usage stuff instead of CC BY-SA though!

**Review by BDpuffy420 — APPROVED** [review]

**Review by BDpuffy420 — APPROVED** [review]

**Pull request merged**